packages/e2e-tests/specs/plugins/custom-taxonomies.test.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	activatePlugin,
+	createNewPost,
+	deactivatePlugin,
+} from '@wordpress/e2e-test-utils';
+
+describe( 'Custom Taxonomies labels are used', () => {
+	beforeAll( async () => {
+		await activatePlugin( 'gutenberg-test-custom-taxonomies' );
+	} );
+
+	beforeEach( async () => {
+		await createNewPost();
+	} );
+
+	afterAll( async () => {
+		await deactivatePlugin( 'gutenberg-test-custom-taxonomies' );
+	} );
+
+	it( 'Ensures the custom taxonomy labels are respected', async () => {
+		const [ button ] = await page.$x( "//button[contains(text(), 'Model') and contains(@class, 'components-button')]" );
+		expect( button ).not.toBeFalsy();
+
+		const labelNew = await page.$x( "//label[@class='components-form-token-field__label' and contains(text(), 'Add New Model')]" );
+		expect( labelNew ).not.toBeFalsy();
+
+		// Open the sidebar.
+		await button.click();
```

Comment:
If I understand correctly, this is to make sure that the custom taxonomy panel is expanded? Since the "Expanded" panels are remembered between page reloads, I might worry that there's a chance this could already be expanded, and proceeding to click the button would _collapse_ the panel.

Would it be possible to check to see that the panel is not already expanded before clicking it?

Reply:
I have check this before clicking on it, is it ok like this ?